docs/views.md

```diff
@@ -0,0 +1,35 @@
+# Views
+
+This page documents how views work in Zulip. If you are new to Django,
+much of this will be familiar, but you may want to read about how
```

Comment:
I think this is flipped? Someone _not_ new to Django would be familiar with how views work, I think. (I assume `If you have experience with Django, much of this` was the intent.)